### PR TITLE
Fix #3369 - Add missing fields to AvailabilityManager:HybridVentilation

### LINKED ISSUE #3369: AvailabilityManager:HybridVentilation missing EnergyPlus fields and OpenStudio methods
In 2.7.0:

AvailabilityManager:HybridVentilation has two numeric fields that aren't implemented in OpenStudio:

* [x] N8 , \field Minimum HVAC Operation Time
      \note Minimum operation time when HVAC system is forced on.
      \type real
      \units minutes
      \minimum 0.0
      \default 0.0
* [x]  N9 ; \field Minimum Ventilation Time
      \note Minimum ventilation time when natural ventilation is forced on.
      \type real
      \units minutes
      \minimum 0.0
      \default 0.0

Other inputs are missing associated SDK methods, though they exist in the OpenStudio.idd:
* [x]  !- Opening Factor Function of Wind Speed Curve Name
* [x]  !- AirflowNetwork Control Type Schedule Name
* [x]  !- Simple Airflow Control Type Schedule Name
* [x]  !- ZoneVentilation Object Name

@jmarrec

### PR DESCRIPTION
**Pull request by jmarrec**
Pull request overview
---------------------

- Fix #3369 
- Add missing fields to AvailabilityManager:HybridVentilation

**Bumped the version to 3.1.0-alpha since I needed VersionTranslation**

### Pull Request Author

Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.

 - [x] Model API Changes / Additions
 - [x] Any new or modified fields have been implemented in the EnergyPlus ForwardTranslator (and ReverseTranslator as appropriate)
 - [x] Model API methods are tested (in `src/model/test`)
 - [x] EnergyPlus ForwardTranslator Tests (in `src/energyplus/Test`)
 - [ ] If a new object or method, added a test in NREL/OpenStudio-resources: Add Link
 - [x] If needed, added VersionTranslation rules for the objects (`src/osversion/VersionTranslator.cpp`)
 - [ ] Checked behavior in OpenStudioApplication, adjusted policies as needed (`src/openstudio_lib/library/OpenStudioPolicy.xml`)
 - [ ] Verified that C# bindings built fine on Windows, partial classes used as needed, etc.
 - [x] All new and existing tests passes
 - [ ] If methods have been deprecated, update rest of code to use the new methods

**Labels:**

 - [x] If change to an IDD file, add the label `IDDChange`
 - [x] If breaking existing API, add the label `APIChange`
 - [x] If deemed ready, add label `Pull Request - Ready for CI` so that CI builds your PR

### Review Checklist

This will not be exhaustively relevant to every PR.
 - [ ] Perform a Code Review on GitHub
 - [ ] Code Style, strip trailing whitespace, etc.
 - [ ] All related changes have been implemented: model changes, model tests, FT changes, FT tests, VersionTranslation, OS App
 - [ ] Labeling is ok
 - [ ] If defect, verify by running develop branch and reproducing defect, then running PR and reproducing fix
 - [ ] If feature, test running new feature, try creative ways to break it
 - [ ] CI status: all green or justified

### CODE REVIEW

File: src/model/AvailabilityManagerHybridVentilation.cpp [hunk 1]
```diff
@@ -115,6 +117,16 @@ namespace detail {
     return result;
   }
 
+  ModelObject AvailabilityManagerHybridVentilation_Impl::clone(Model model) const {
+    auto avmClone = AvailabilityManager_Impl::clone(model).cast<AvailabilityManagerHybridVentilation>();
+
+    // Makes little sense to me to keep these two which are AirLoopHVAC specific. If you're cloning this object, it's to set to to ANOTHER AirLoopHVAC
```
Comment (kbenne) [inline src/model/AvailabilityManagerHybridVentilation.cpp:123]: Agreed. This seems like the correct thing to do.

**Comment by kbenne** [comment]
I'm good with this. The testing is impressively thorough. I would just like to make sure @tijcolem is ready to accept the version bump.

Is there an integration test @jmarrec?

**Comment by jmarrec** [comment]
@kbenne I did not add an integration test no.

That would need to be a fully new test for the three existing fields that didn't have an implementation, and these fields are 1) weird and not straightforward (I'm not 100% sure how to produce a valid test, so it's going to require time and effort), 2) mutually exclusive. 3) this object is not one of the most common ones. 4) the existing test for AVMs is very minimal https://github.com/NREL/OpenStudio-resources/blob/a729a7c85f55cc05d7ba6d50c45d21b8621611c9/model/simulationtests/availability_managers.rb#L48-L49. 

Instead I tested a **bunch** here. 

I can add one if you really want to though.

**Comment by jmarrec** [comment]
https://github.com/NREL/OpenStudio/pull/4037/commits/99aa952455a3976dfa5ea34a3c16ccabfdb5c6ef re-nables `-Wall` on Unix, and fixes #4038

**Comment by jmarrec** [comment]
Linux and mac are happy, aside from the usual suspect:

```
The following tests FAILED:
	614 - BCLFixture.RemoteBCLTest (SIGTRAP)
	616 - BCLFixture.GetComponentByUID (SIGTRAP)
```

Ready to drop in. (I need the VT part for another PR, so better sooner than later)

@kbenne do I need to add an OpenStudio-resources test?

**Comment by tijcolem** [comment]
Good with version bump!

**Comment by jmarrec** [comment]
Windows seem to have failed with the same error as on #4029 

```D:\jenkins\openstudio\develop\build\src/cli/embedded_files/ruby/2.5.0/gems/openstudio-standards-0.2.11/data/weather/YUBA-CO_724838_CZ2010_epw.cxx(1): fatal error C1060: compiler is out of heap space (compiling source file D:\jenkins\openstudio\develop\build\src\cli\embedded_files.cxx) [D:\jenkins\openstudio\develop\build\src\cli\openstudio.vcxproj]
```

@tijcolem https://ci.commercialbuildings.dev/blue/organizations/jenkins/openstudio-incremental-windows/detail/PR-4037/5/pipeline#step-34-log-99

**Pull request merged**